package.json

```diff
@@ -54,6 +54,8 @@
     "commit": "git-cz"
   },
   "dependencies": {
+    "@devlaunchers/components": "workspace:*",
+    "@radix-ui/react-checkbox": "1.0.4",
```

Comment:
you installed these `"@devlaunchers/components"` and `"@radix-ui/react-checkbox"` globally to whole monorepo. the root `package.json` should only include deps that will manage the whole experience of the monorepo, for example `rimraf` is installed globally because sometimes someone might want to delete every `node_modules` folder in the monorepo with the script `yarn nuke:node_modules` that can be found in the `package.json` in the root of the monorepo.

to fix this we need to remove `"@devlaunchers/components"` and `"@radix-ui/react-checkbox"` from the global `package.json` then run `yarn`